kustomize/resource_kustomization.go

```diff
@@ -48,7 +48,7 @@ func kustomizationResourceCreate(d *schema.ResourceData, m interface{}) error {
 	srcJSON := d.Get("manifest").(string)
 	u, err := parseJSON(srcJSON)
 	if err != nil {
-		return fmt.Errorf("ResourceCreate: %s", err)
+		return fmt.Errorf("ResourceCreate: JSON parse error: '%s' %s", srcJSON, err)
```

Comment:
I'm not sure outputting all srcJSON is a good idea. The resource may be a secret and now credentials are in some CI/CD log. I have to admit, I don't even know what err includes in this case.